addons/mail/models/ir_config_parameter.py

```diff
@@ -1,21 +1,35 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
 
 
 class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
+    def _check_mail_catchall_allowed_domains(self, value):
```

Comment:
:socks: would define it after the main methods (aka after create / write), as CRUD methods should generally be on top of the file :)